addons/website_event/data/website_snippet_data.xml

```diff
@@ -6,7 +6,10 @@
         <field name="name">Upcoming Events</field>
         <field name="model_id">event.event</field>
         <field name="user_id" eval="False" />
-        <field name="domain">[('date_begin', '&gt;=', context_today())]</field>
+        <field name="domain">[
+            ('date_begin', '&gt;=', context_today()),
+            ('can_see_on_website', '=', True)
```

Comment:
Don't we want to apply this on our new "Upcoming and Ongoing Events" snippet as well?